.github/workflows/ci.yml

```diff
@@ -32,7 +43,7 @@ jobs:
       - uses: actions/download-artifact@v2
         with:
           name: cardano-wallet-tests-win64
-      - run: '.\\cardano-wallet-core-test-unit.exe --color'
+      - run: '.\\cardano-wallet-core-test-unit.exe --color +RTS -N1'
```

Comment:
:crossed_fingers: didn't help locally for me. Let's see here :man_shrugging:

Reply:
OK https://github.com/input-output-hk/cardano-wallet/pull/2527#issuecomment-780439348 I believe you :+1:

I will change this to `-N` to override the `ghc -with-rtsopts=-N4` setting from cabal. Then it should set the number of threads to match [the VM specs](https://docs.github.com/en/actions/reference/specifications-for-github-hosted-runners#supported-runners-and-hardware-resources).

Reply:
Sounds like getting rid of parallel didn't help :( 
We may have some property tests that are a bit too much for Windows?

Reply:
Something odd is going on with RTS opts. It's supposed to have `-M2G` set for both windows and linux.

Reply:
Something I've experienced actually with specifying rts opts directly in the cabal file is that:

```
      "-with-rtsopts=-M2G"
      "-with-rtsopts=-N4"
```

This doesn't work. Only the last option gets taken into account. But this does:

```
      - '"-with-rtsopts=-M2G -N4"'
```